code/game/turfs/simulated/walls_misc.dm

```diff
@@ -53,10 +53,47 @@
 
 /turf/simulated/wall/shuttle
 	name = "wall"
-	icon = 'icons/turf/shuttle.dmi'
-	icon_state = "wall1"
+	icon = 'icons/turf/walls/shuttle_wall.dmi'
+	icon_state = "shuttle"
 	walltype = "shuttle"
-	smooth = SMOOTH_FALSE
+	smooth = SMOOTH_MORE|SMOOTH_DIAGONAL
+	canSmoothWith = list(/turf/simulated/wall/shuttle, /obj/structure/window/shuttle)
```

Comment:
should also smooth with shuttle engines and shuttle doors/glass airlocks, so that current shuttle appearances are preserved.